tests/test_release_gates_do_not_block_good_documents.py

```
@@ -0,0 +1,223 @@
+def test_corpus_path_is_configurable(monkeypatch, tmp_path):
+    """Корпус можно положить на постоянный том вместо эфемерного контейнера.
+
+    Без этого каждый рестарт собирал корпус с adilet заново, и неудачная
+    загрузка оставляла сервис вообще без норм — а значит без документов.
+    """
+    import importlib
+
+    monkeypatch.setenv("KORGAN_CORPUS_DB", str(tmp_path / "persistent" / "corpus.sqlite3"))
+    module = importlib.reload(importlib.import_module("korgan.legal.corpus"))
+    try:
+        assert module.DEFAULT_DB_PATH == tmp_path / "persistent" / "corpus.sqlite3"
```

Ruff (0.16.2)

[warning] Yoda condition detected

(SIM300)